README.md

```diff
@@ -1,26 +1,28 @@
 # GitHub OAuth Ruby Quickstart
+
+[DEGW: An intro here motivating the problem you are trying to help solve. Explain (briefly) what an OAuth app is, and why you might want to build one. Why do we need a guide? Is it hard to do? (Narrator: In fact, it is hard.) ALso give a preview of what the reader can expect to have achieved by the end of the article.]
```

Comment:
This is definitely necessary. I think the addressing the motivation of this quickstart would do really well combined with an Intro to OAuth 101 article / post, which would link out to the various language / framework specific quickstarts (the first of which is here). That way the "Intro to the OAuth dance /process + Motivations and outcomes of using OAuth" would live together in a single post which would be a prerequisite of these technical quickstarts.